pygments_routerlexers/Cisco/IOS.py

```diff
@@ -10,9 +10,9 @@ class IOSLexer(RegexLexer):
         "root" : [
             (r"^!.*", Comment),
             (r"^(version\s+)(.*)$", bygroups(Keyword, Number.Float)),
-            (r"(description)(.*?)$", bygroups(Keyword,Comment)),
+            (r"(desc*r*i*p*t*i*o*n*)(.*?)$", bygroups(Keyword,Comment)),
```

Comment:
Yeah, this matches `des` or any more complete version. I tend to use something in between when I'm giving commands, such as `'inter` rather than just `int`. The way you suggested with example input:

`descr Test`

would put `desc` as the first capture, `r Test` as the second